go/value_wrapper.go

```diff
@@ -238,3 +241,120 @@ func (valueWrapper ValueWrapper) GetType() string {
 	}
 	return "empty"
 }
+
+/*
+String() returns the value in the ValueWrapper as a string.
+Maps in the output will be sorted by key value in alphabetical order.
+For vetex, the output is in form (vid: tagName{propKey: propVal, propKey2, propVal2}),
+For edge, the output is in form (SrcVid)-[name]->(DstVid)@Ranking{prop1: val1, prop2: val2}
+where arrow direction depends on edgeType
+For path, the output is in form(v1)-[name@edgeRanking]->(v2)-[name@edgeRanking]->(v3)
+*/
+func (valWarp ValueWrapper) String() string {
+	value := valWarp.value
+	if value.IsSetNVal() {
+		return value.GetNVal().String()
+	} else if value.IsSetBVal() {
+		return fmt.Sprintf("%t", value.GetBVal())
+	} else if value.IsSetIVal() {
+		return fmt.Sprintf("%d", value.GetIVal())
+	} else if value.IsSetFVal() {
+		fStr := strconv.FormatFloat(value.GetFVal(), 'f', -1, 64)
+		if !strings.Contains(fStr, ".") {
+			fStr = fStr + ".0"
+		}
+		return fStr
+	} else if value.IsSetSVal() {
+		return `"` + string(value.GetSVal()) + `"`
+	} else if value.IsSetDVal() {
+		return value.GetDVal().String()
```

Comment:
why the Date can directly to call it's String()

Reply:
fixed. was thinking to add String() for time/date/dateTime when we have wrappers for these values